### PR TITLE
helm: workflow_dispatch use commit sha

### PR DESCRIPTION
**Pull request by ibiqlik**

### CODE REVIEW

File: action-helm-tools/main.sh [hunk 1]
```diff
@@ -31,9 +31,16 @@ set_commit_status() {
   STATUS=$1
 
   if [[ "$GITHUB_EVENT_NAME" == "workflow_dispatch" ]] && [[ -n "$GITHUB_SHA" ]] && [[ -n "$GITHUB_TOKEN" ]] && [[ -n "$GITHUB_REPOSITORY" ]]; then
+
+    if [ -n "$COMMITSHA" ]; then
```
Comment (ibiqlik) [inline action-helm-tools/main.sh:35]: If there is a "commitsha" workflow input we will use that to set commit status to

File: scripts-circleci/README.md [hunk 2]
```diff
@@ -11,3 +11,52 @@ Place the following snippet in your CircleCI config to call `package-helm` Githu
...
+name: Package Helm chart
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version used in Docker image tag and Helm chart'
+        required: true
+      commitsha:
+        description: 'Commit SHA'
+        required: false
```
Comment (ibiqlik) [inline scripts-circleci/README.md:27]: New input

File: scripts-circleci/README.md [hunk 3]
```diff
@@ -11,3 +11,52 @@ Place the following snippet in your CircleCI config to call `package-helm` Githu
...
+  package-chart:
+    name: Package Helm chart
+    runs-on: ubuntu-latest
+    steps:
+      # Prints Action envrionment information useful for debugging
+      - uses: qlik-oss/ci-tools/action-print-event-info@master
+
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.commitsha }}
```
Comment (ibiqlik) [inline scripts-circleci/README.md:46]: Here we checkout the commit that comes as an input (usually from circleci as CIRCLE_SHA1)

File: scripts-circleci/github-package-helm-dispatch.sh [hunk 4]
```diff
@@ -18,6 +18,12 @@ if [ -z "${VERSION}" ]; then
   exit 1
 fi
 
+# If VERSION is SEMVER set TAG_REF for workflow_dispatch
+if echo "${VERSION#v}" | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$'; then
+    echo "TAG_REF=v${VERSION#v}"
+    TAG_REF="v${VERSION#v}"
+fi
```
Comment (ibiqlik) [inline scripts-circleci/github-package-helm-dispatch.sh:25]: When cloud releaser used there is no TAG trigger so check if the version is semver then use that as the REF

**Review by qliklars — APPROVED** [review]
General comment is that I think this is starting to be too much code for bash scripts,
but that is an irrelevant comment for an emergency fix

**Review by mtk7801 — APPROVED** [review]

**Pull request merged**